### PR TITLE
Plugins info in readme and tutorial; questions header in readme

### PR DESCRIPTION
**Pull request by idk3**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -41,6 +41,11 @@ Authors
 The first release of FermiLib (v0.1a0) was developed by `Ryan Babbush <https://research.google.com/pubs/RyanBabbush.html>`__, `Jarrod McClean <https://crd.lbl.gov/departments/computational-science/ccmc/staff/alvarez-fellows/jarrod-mcclean/>`__, `Damian S. Steiger <http://www.comp.phys.ethz.ch/people/person-detail.html?persid=165677>`__, `Ian D. Kivlichan <http://aspuru.chem.harvard.edu/ian-kivlichan/>`__, `Thomas
 Häner <http://www.comp.phys.ethz.ch/people/person-detail.html?persid=179208>`__, `Vojtech Havlicek <https://github.com/VojtaHavlicek>`__, `Matthew Neeley <https://maffoo.net/>`__, and `Wei Sun <https://github.com/Spaceenter>`__.
 
+Questions?
+----------
+
+Contact fermilib at projectq.ch.
```
Comment (babbush) [inline README.rst]: I would write: "Contact the FermiLib team by emailing us at [fermilib@projectq.ch](fermilib@projectq.ch)"

So actually include the link. These days, bots are smart enough to look for " at " and identify it as an email. I'm not that worried.

File: README.rst [hunk 2]
```diff
@@ -20,7 +20,12 @@ Getting started
 ---------------
 
 To start using FermiLib, simply follow the installation instructions in the `intro <http://fermilib.readthedocs.io/en/latest/intro.html>`__. There, you will also find `code examples <http://fermilib.readthedocs.io/en/latest/examples.html>`__. Also, make sure to check out the `ProjectQ
-website <http://www.projectq.ch>`__ and the detailed `code documentation <http://fermilib.readthedocs.io/en/latest/fermilib.html>`__. Moreover, take a look at the available `plugins <http://projectq.ch/code-and-docs/#Fermilib>`__ for FermiLib.
+website <http://www.projectq.ch>`__ and the detailed `code documentation <http://fermilib.readthedocs.io/en/latest/fermilib.html>`__. Moreover, take a look at the available plugins for FermiLib.
+
+Plugins
+-------
+
+In order to generate molecular hamiltonians in Gaussian basis sets and perform other complicated electronic structure calculations, one can install plugins. We currently support PySCF and Psi4, plugins for which can be found `here <http://projectq.ch/code-and-docs/#Fermilib>`__.
```
Comment (babbush) [inline README.rst]: No, please link directly to the git repos for Psi4 and pyscf plugins.

File: README.rst [hunk 3]
```diff
@@ -41,6 +46,11 @@ Authors
 The first release of FermiLib (v0.1a0) was developed by `Ryan Babbush <https://research.google.com/pubs/RyanBabbush.html>`__, `Jarrod McClean <https://crd.lbl.gov/departments/computational-science/ccmc/staff/alvarez-fellows/jarrod-mcclean/>`__, `Damian S. Steiger <http://www.comp.phys.ethz.ch/people/person-detail.html?persid=165677>`__, `Ian D. Kivlichan <http://aspuru.chem.harvard.edu/ian-kivlichan/>`__, `Thomas
 Häner <http://www.comp.phys.ethz.ch/people/person-detail.html?persid=179208>`__, `Vojtech Havlicek <https://github.com/VojtaHavlicek>`__, `Matthew Neeley <https://maffoo.net/>`__, and `Wei Sun <https://github.com/Spaceenter>`__.
 
+Questions?
+----------
+
+If you have any other questions, please contact fermilib at projectq.ch.
```
Comment (damiansteiger) [inline README.rst]: Could you display the full email address as `fermilib@projectq.ch`?
Makes it simple to copy & paste (from our previous experience it does not lead to more spam)

Comment (babbush) [inline README.rst]: Yeah I made the same comment but somebody *cough* @idk3 *cough* was worried that bots would find the email address and spam us. But I don't think it is too common to crawl GitHub for this sort of thing. Ian, please change (or contest) and then write something so I know you made the change. I will then merge the PR.

Comment (idk3) [inline README.rst]: loooool ok, I've changed it

**Pull request merged**